library-android/src/main/java/nl/elements/objectstore/android/stores/PreferencesStore.kt

```diff
@@ -117,10 +117,11 @@ private class StorePreferencesEditor(
 
     override fun putFloat(key: String?, value: Float): SharedPreferences.Editor = put(key, value)
 
-    override fun putString(key: String?, value: String?): SharedPreferences.Editor = put(key, value)
+    override fun putString(key: String?, value: String?): SharedPreferences.Editor = put(key, value!!)
```

Comment:
Can't value be non null then? Otherwise don't force unwrap?

Reply:
At the moment the lib anticipates a value to be non-null whenever it got requested. When requesting you always should check wether there is a value first.

I got some feedback on the and I would like to refactor to 'Maybe'.